index.bs

```diff
@@ -5950,8 +5945,9 @@ partial interface MLGraphBuilder {
     function where(builder, condition, input, other) {
       const c = builder.clamp(condition, {'minValue': 0, 'maxValue': 1});
       builder.add(
-        builder.mul(input, builder.cast(c, input.dataType())),
-        builder.mul(other, builder.cast(builder.logicalNot(c), other.dataType())));
```

Comment:
The above two lines were good and won't need to change.

Reply:
Oops. :) Thanks.